vmdb/app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -46,7 +46,7 @@ def textual_type
   end
 
   def textual_port
-    ["openstack", "openstack_infra", "rhevm"].include?(@ems.emstype) ? {:label => "API Port", :value => @ems.port} : nil
+    @ems.class.supports_port? ? {:label => "API Port", :value => @ems.port} : nil
```

Comment:
Now that I reviewed all the callers of the `supports_port?` method, I am questioning why we have it at the class level at all?  All the callers take an ems and do `ems.class.supports_port?`.  That seems wrong.